### DISCUSSION TITLE
Cyberduck connections blocked my McAfee LiveSafe

### DISCUSSION THREAD

**Discussion by csuebws** [iterate-ch/cyberduck#15305] [Q&A]
What steps do I need to take, or which protocols or rules do I need to enable to authorize connections with McAfee LiveSafe? Thanks

**Comment by ylangisc** [comment]
There is no general rule here as it depends on the protocol you use (WebDAV, SFTP, ...). No idea how LiveSafe works but I guess you should see what applications and connections are blocked and can then unblock them.
